lib/github_api/client/projects/columns.rb

```diff
@@ -3,10 +3,15 @@
 
 module Github
   class Client::Projects::Columns < API
-    PREVIEW_MEDIA = "application/vnd.github.inertia-preview+json" # :nodoc:
     REQUIRED_COLUMN_PARAMS = %w(name).freeze
     REQUIRED_MOVE_COLUMN_PARAMS = %w(position).freeze
 
+    require_all 'github_api/client/projects/columns',
+                'cards'
+
+    # Access to Projects::Cards API
+    namespace :cards
```

Comment:
@piotrmurach I mirrored the github API here. It seems to treat cards as children of columns, and requires a column ID. You cannot list all the cards for a project.